handlers/main.yml

```diff
@@ -1,6 +1,10 @@
 # SPDX-License-Identifier: MIT
 ---
-- name: restart firewalld
+# Put handlers for tasks here.
+
+# Example:
+- name: template handler to restart services
   service:
-    name: firewalld
+    name: "{{ item }}"
     state: restarted
+  loop: "{{ __template_services }}"
```

Comment:
this should be `__mssql_services`

Reply:
The MVP role will not use handlers. I will remove this file when the role is ready.

Reply:
> The MVP role will not use handlers. I will remove this file when the role is ready.

Why won't it use handlers?  Won't the role need to restart the mssql service if something has changed?

Reply:
mssql-conf takes care of the service. When you run mssql-conf to setup mssql server, it starts and enables the mssql-server service. For the MVP, no other actions other than setup are needed.